### PR TITLE
Add menu item to copy Node URL in RM

### PR DESCRIPTION
**Pull request by jrochas**

### CODE REVIEW

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 6 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [ContextMenu.java#L43](https://github.com/ow2-proactive/scheduling-portal/blob/122fde64cd04c37e9f1be90b002b733d832d6d53/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/ContextMenu.java#L43): This class has 9 parents which is greater than 5 authorized. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AMaximumInheritanceDepth)
1. ![MAJOR][MAJOR] [ContextMenu.java#L181](https://github.com/ow2-proactive/scheduling-portal/blob/122fde64cd04c37e9f1be90b002b733d832d6d53/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/ContextMenu.java#L181): Define and throw a dedicated exception instead of using a generic one. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)
1. ![MINOR][MINOR] [ContextMenu.java#L26](https://github.com/ow2-proactive/scheduling-portal/blob/122fde64cd04c37e9f1be90b002b733d832d6d53/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/ContextMenu.java#L26): Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)
1. ![MINOR][MINOR] [ContextMenu.java#L43](https://github.com/ow2-proactive/scheduling-portal/blob/122fde64cd04c37e9f1be90b002b733d832d6d53/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/ContextMenu.java#L43): Override the "equals" method in this class. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2160)
1. ![MINOR][MINOR] [ContextMenu.java#L323](https://github.com/ow2-proactive/scheduling-portal/blob/122fde64cd04c37e9f1be90b002b733d832d6d53/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/ContextMenu.java#L323): Define a constant instead of duplicating this literal "To File" 3 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ContextMenu.java#L324](https://github.com/ow2-proactive/scheduling-portal/blob/122fde64cd04c37e9f1be90b002b733d832d6d53/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/ContextMenu.java#L324): Define a constant instead of duplicating this literal "To Catalog" 3 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/ContextMenu.java [hunk 1]
```diff
@@ -127,6 +130,8 @@ public static Menu createContextMenuFromCompactView(RMController controller, Obj
                 lockItem.setEnabled(true);
                 unlockItem.setEnabled(false);
             }
+        } else {
+            copyItem.setEnabled(false);
```
Comment (chloegugli) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/ContextMenu.java]: You don't need to set this to true anywhere?

Comment (jrochas) [reply rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/ContextMenu.java]: Normally no, it's enabled by default, but you're right, to be consistent with the other items I'll enable it back in the other case.

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 6 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [ContextMenu.java#L43](https://github.com/ow2-proactive/scheduling-portal/blob/02c010f203ca1e5b158d958f9dc820db723a8a8a/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/ContextMenu.java#L43): This class has 9 parents which is greater than 5 authorized. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AMaximumInheritanceDepth)
1. ![MAJOR][MAJOR] [ContextMenu.java#L189](https://github.com/ow2-proactive/scheduling-portal/blob/02c010f203ca1e5b158d958f9dc820db723a8a8a/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/ContextMenu.java#L189): Define and throw a dedicated exception instead of using a generic one. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)
1. ![MINOR][MINOR] [ContextMenu.java#L26](https://github.com/ow2-proactive/scheduling-portal/blob/02c010f203ca1e5b158d958f9dc820db723a8a8a/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/ContextMenu.java#L26): Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)
1. ![MINOR][MINOR] [ContextMenu.java#L43](https://github.com/ow2-proactive/scheduling-portal/blob/02c010f203ca1e5b158d958f9dc820db723a8a8a/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/ContextMenu.java#L43): Override the "equals" method in this class. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2160)
1. ![MINOR][MINOR] [ContextMenu.java#L365](https://github.com/ow2-proactive/scheduling-portal/blob/02c010f203ca1e5b158d958f9dc820db723a8a8a/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/ContextMenu.java#L365): Define a constant instead of duplicating this literal "To File" 3 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ContextMenu.java#L366](https://github.com/ow2-proactive/scheduling-portal/blob/02c010f203ca1e5b158d958f9dc820db723a8a8a/rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/ContextMenu.java#L366): Define a constant instead of duplicating this literal "To Catalog" 3 times. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**